<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FacilityEditor</class>
 <widget class="QWidget" name="FacilityEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="1">
      <widget class="QSpinBox" name="sbCapacity">
       <property name="maximum">
        <number>99999</number>
       </property>
       <property name="singleStep">
        <number>500</number>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="2">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Спортивные сооружения</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Тип сооружения</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="cbType"/>
     </item>
     <item row="3" column="0" colspan="2">
      <widget class="QTableView" name="tableView"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Вместительность, не менее</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QPushButton" name="pbnExec">
     <property name="text">
      <string>Выполнить</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
